README.md

```diff
@@ -56,36 +60,30 @@ Converting RTL into DRC clean GDS takes less than 10 lines of simple Python code
 ```python
 import siliconcompiler                             # import python package
 chip = siliconcompiler.Chip('heartbeat')           # create chip object
-chip.load_target('freepdk45_demo')                 # load a pre-defined target
-chip.set('input', 'rtl', 'verilog', 'heartbeat.v') # set input sources
-chip.set('input', 'asic', 'sdc', 'heartbeat.sdc')  # set constraints
-#chip.set('option','remote', True)                 # enable remote execution
+chip.load_target('skywater130_demo')               # load a pre-defined target
+chip.input('heartbeat.v')                          # set input sources
+chip.clock('clk', period=10)                       # set constraints
+chip.set('option','remote', True)                  # enable remote execution
 chip.run()                                         # run compilation
 chip.summary()                                     # print summary
 chip.show()                                        # show layout
 ```
 
-To reduce the pain of tool installation, the project supports free remote compilation at [siliconcompiler.com](siliconcompiler.com).
-
-1. Sign up for a [free beta account](https://www.siliconcompiler.com/beta),
-2. Create a [credentials file](https://docs.siliconcompiler.com/en/latest/user_guide/installation.html#cloud-access)
-3. Set the remote option to True (see example above)
-4. Run
+To reduce the pain of tool installation, the project supports free remote compilation at [siliconcompiler.com](https://www.siliconcompiler.com).
 
 Simple designs can be compiled using the built in command line 'sc' app:
 
 ```sh
-sc -remote heartbeat.v -design heartbeat -target "freepdk45_demo"
+sc -remote -target "asic_demo"
 ```
 
 # Documentation
 
-The full reference manual and tutorials can be found [HERE](https://docs.siliconcompiler.com/en/latest/).
+The full reference manual and tutorials can be found [HERE](https://docs.siliconcompiler.com/).
 
 # Installation
 
-
-Complete installation instructions are available in the [Installation Guide](https://docs.siliconcompiler.com/en/latest/user_guide/installation.html).
+Complete installation instructions are available in the [Installation Guide](https://docs.siliconcompiler.com/en/stable/user_guide/installation.html).
 
 To install the project from source (recommended for developers only).
```

Comment:
I'm okay with the install code block as is, but just want to make a note:
This is repeating instructions from the latest version of the installation docs. You could directly point to the developer installation docs here: https://docs.siliconcompiler.com/en/latest/user_guide/installation.html#install-from-github-repo-linux-macos
But right now, that link will only exist on the "latest" docs and if you point to "stable" it will be a broken link.